src/App.js

```diff
@@ -1,40 +1,75 @@
-import React from "react";
-import "./App.css";
-import { Container, Grid } from "@material-ui/core/";
-import Header from "./components/Main/Header";
-import { makeStyles } from "@material-ui/core/styles";
 
+import React, { useState } from 'react';
+import './App.css';
+import { Container, Grid } from '@material-ui/core/';
+import { makeStyles } from '@material-ui/core/styles';
+import URLInput from './components/URLInput/URLInput';
+import DisplayResults from './components/Results/displayResults';
 
+//styles
 const useStyles = makeStyles({
-  containerStyle: {
-    boxShadow: "8px 8px #d9d9d9",
-    background: "#ffffff",
+  containerStyle : {
+    boxShadow: '8px 8px #d9d9d9',
+    background: '#ffffff',
   },
 
   footer: {
-    background: "#6bb26d",
-    padding: "7px",
-  },
-});
+        background: '#6bb26d',
+        padding: '7px',
+  }
+})
+
+//getUrl
+ const inputData = {
+   url: "",
+ };
+
+ const getURL = (text) => {
+   inputData.url = text;
+ }
 
 function App() {
-  const classes = useStyles();
+const classes = useStyles();
+
+ //request
+ const [request, setRequest] = useState("");
 
   return (
     <div className="App">
       <Container fixed>
-        <div className={classes.containerStyle}>
-          <Header />
-          
-          <Grid container className={classes.footer}>
-            <Grid item xs={12}>
-              <h5>The Sentimentalists &copy; 2020</h5>
-            </Grid>
-          </Grid>
-        </div>
-      </Container>
+
+    <div> 
```

Comment:
@GCJyo Can we have a chat about this bit? Just need to clarify a couple of things before we approve the pull request.